packages/dm-core-plugins/src/form/components/AttributeList.tsx

```diff
@@ -0,0 +1,40 @@
+import { Stack, TAttribute, TBlueprint } from '@development-framework/dm-core'
+import React from 'react'
+import { AttributeField } from '../fields/AttributeField'
+import { TConfig } from '../types'
+
+export const AttributeList = (props: {
```

Comment:
The content of this function is identical to the code block removed from ObjectField, so I wouldn't spend time reviewing it